__init__.py

```diff
@@ -17,13 +17,16 @@
 from uuid import uuid4
 from requests import HTTPError
 from adapt.intent import IntentBuilder
```

Comment:
PEP8 says there should be a blank line between built-in imports and third party imports.  it also says there should be a blank line between third party imports and local package imports (Mycroft, in this case)